zeroex/ordervalidator/order_validator.go

```diff
@@ -357,6 +357,7 @@ func (o *OrderValidator) BatchValidate(ctx context.Context, rawSignedOrders []*z
 						log.WithFields(log.Fields{
 							"error":     err.Error(),
 							"numOrders": len(trimmedOrders),
+							"orders":    trimmedOrders,
```

Comment:
@jalextowle [There were about 1,600 occurrences](https://kibana.mesh.0x.org/goto/27e67dfdaf42866e3131acbcd8359ad8) of this warning over the past 24 hours. Is this going to result in too much data in the logs?

Reply:
Yeah, let's remove this. This warning occurs a lot. Anything that causes the RPC request to fail, including network issues will trigger it. Would orderHashes suffice? If it's necessary for debugging, can we be more judicious by introspecting the returned error and only throw if it's a smart contract revert reason?

Reply:
No, order hashes don't help. The only thing that is useful is the contents of the order. I can do the introspection instead.